strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/MavenRepositoryManagementServiceImplTest.java

```diff
@@ -47,14 +48,17 @@
 
     private static final String REPOSITORY_RELEASES_MERGE_2 = "rmsi-releases-merge-2";
 
-    @org.springframework.context.annotation.Configuration
-    @Import({ CommonConfig.class,
-              StorageCoreConfig.class,
-              Maven2LayoutProviderConfig.class })
-    public static class SpringConfig
-    {
+    private Repository repository_release_1;
```

Comment:
What is this here about? I can't see these variables as being used anywhere. Am I missing something?